tutorials/colors/colormapnorms.py

```diff
@@ -226,49 +226,29 @@
 
 
 ###############################################################################
-# Custom normalization: Two linear ranges
-# ---------------------------------------
+# Custom normalization: Manually impliment two linear ranges
+# ----------------------------------------------------------
 #
-# It is possible to define your own normalization.  In the following
-# example, we modify :func:`colors:SymLogNorm` to use different linear
-# maps for the negative data values and the positive.  (Note that this
-# example is simple, and does not validate inputs or account for complex
-# cases such as masked data)
-#
-# .. note::
-#    This may appear soon as :func:`colors.OffsetNorm`.
-#
-#    As above, non-symmetric mapping of data to color is non-standard
-#    practice for quantitative data, and should only be used advisedly.  A
-#    practical example is having an ocean/land colormap where the land and
-#    ocean data span different ranges.
-
-N = 100
-X, Y = np.mgrid[-3:3:complex(0, N), -2:2:complex(0, N)]
-Z1 = np.exp(-X**2 - Y**2)
-Z2 = np.exp(-(X - 1)**2 - (Y - 1)**2)
-Z = (Z1 - Z2) * 2
-
+# The `.DivergingNorm` described above makes a useful example for
+# defining your own norm.
 
 class MidpointNormalize(colors.Normalize):
-    def __init__(self, vmin=None, vmax=None, midpoint=None, clip=False):
-        self.midpoint = midpoint
+    def __init__(self, vmin=None, vmax=None, vcenter=None, clip=False):
```

Comment:
Wait, how is that different from divergingnorm at that point (if any)?  Would it be worth changing it to showcase something else, or just delete this?

Reply:
No difference.  But its useful to show how to make your own norm; if someone wants a new PR with a new norm, they should totally do that...

Reply:
Maybe you should mention that this does the same as `DivergingNorm` but is here as an example to illustrate creating your own norm.